internal/logd/logd.go

```diff
@@ -107,9 +107,14 @@ func syslogToEvent(syslogMessage syslog.Message) (*api.AddEventInput, error) {
 		}
 	}
 
+	message := *rfc5425Message.Message
+	if message[len(message)-1] == '\n' {
```

Comment:
`message = strings.TrimSuffix(message, "\n")`

Reply:
especially because if this is possibly false, then it implies that message == "" is possibly true, which means this will be an index-out-of-bounds panic

Reply:
Ah! That's what I was looking for! I only found `TrimRight`. Thanks!